### PR TITLE
Update: ignore query part of url to support gulp-rev-append

### PR DESCRIPTION
**Pull request by valorkin**
@jonkemp we spoke it someday,
it became critical again after clean-css update
if you have something against block scoped vars
or any I can apply 

[Review on Reviewable](https://reviewable.io/reviews/jonkemp/gulp-useref/84)

### CODE REVIEW

**Comment by valorkin** [comment]
@jonkemp any better ideas?

**Comment by jonkemp** [comment]
Linking the original pull request here for reference.

#71

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```
Comment (jonkemp) [inline .gitignore]: This is not needed. You can add to your local .gitignore if you want to always ignore that.

**Comment by valorkin** [comment]
@jonkemp .idea removed

**Comment by valorkin** [comment]
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/jonkemp/gulp-useref/84)



---

@jonkemp can upi apply it please, may be I will have to do one more PR, because this module is not working on heroku at all

---

**Comment by jonkemp** [comment]
Sorry for leaving this open for so long, but I'm closing now. I'm not interested in this at this time. Please refer to the Contributing doc.

https://github.com/jonkemp/gulp-useref/blob/master/CONTRIBUTING.md#pull-requests

Before submitting a pull request that fixes a problem you have, please see if anyone has the issue. If not, create an issue or a feature request first before submitting a pull request. Thanks!

**Comment by valorkin** [comment]
@jonkemp switched to webpack, so it is completely not relevant :)

**Pull request closed without merging**